scanpipe/models.py

```diff
@@ -657,9 +658,13 @@ def get_latest_output(self, filename):
 
     def walk_codebase_path(self):
         """
-        Returns all files and directories path of the codebase/ directory recursively.
+        Returns all files and directories path of the codebase/ directory recursively,
+        including the codebase/ directory itself.
         """
-        return self.codebase_path.rglob("*")
+        return itertools.chain(
```

Comment:
@tdruez I am trying to get `walk_codebase_path()` to return the `codebase/` directory in addition to the subdirectories and files of `codebase/`. Is there a better way to do this?